web/custom-plugins/for-operations/src/Enums.ts

```diff
@@ -1,11 +0,0 @@
-export enum OperationStatus {
-  UPCOMING = 'Upcoming',
-  IN_PROGRESS = 'In progress',
```

Comment:
Not sure we need to delete the old for-operation stuff.
They could be used as a basis for future development

Reply:
Yeah, agreed! Will revert those deletions